packages/openapi-v3/test/unit/controller-spec/controller-decorators/param-decorators/param-path.test.ts

```diff
@@ -71,19 +75,23 @@ describe('Routing metadata for parameters', () => {
     it('defines a parameter with in:path type:boolean', () => {
       class MyController {
         @get('/greet/{name}')
-        @param.path.boolean('name')
-        greet(name: string) {}
+        greet(@param.path.boolean('name') name: boolean) {}
       }
 
       const actualSpec = getControllerSpec(MyController);
 
       expect(actualSpec.paths['/greet/{name}']['get'].parameters).to.eql([
         {
           name: 'name',
-          type: 'boolean',
           in: 'path',
+          schema: {
+            type: 'boolean',
+          },
         },
       ]);
     });
   });
+  // For review purpose:
+  // If we agree to add other shortcuts, I am adding them then.
+  // 3 javascript primitive types + 9 OpenAPI primative types in total
```

Comment:
I think those other shortcuts are out of scope of MVP, but since you already have them implemented, then I don't mind adding them as part of this pull request. Whatever makes more sense to you.